chromium_src/chrome/common/webui_url_constants.cc

```diff
@@ -4,17 +4,26 @@
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 #include "brave/components/constants/webui_url_constants.h"
+#include "brave/components/ipfs/buildflags/buildflags.h"
 #include "chrome/common/webui_url_constants.h"
 
-#define kChromeUIAttributionInternalsHost                                    \
-  kChromeUIAttributionInternalsHost, kAdblockHost, kAdblockInternalsHost,    \
-      kIPFSWebUIHost, kRewardsPageHost, kRewardsInternalsHost, kWelcomeHost, \
-      kWalletPageHost, kTorInternalsHost
+#define kChromeUIAttributionInternalsHost                                     \
+  kChromeUIAttributionInternalsHost, kAdblockHost, kAdblockInternalsHost,     \
+      kRewardsPageHost, kRewardsInternalsHost, kWelcomeHost, kWalletPageHost, \
+      kTorInternalsHost
 #define kChromeUIPerformanceSettingsURL kChromeUIPerformanceSettingsURL_UnUsed
 #define kPerformanceSubPage kPerformanceSubPage_UnUsed
 
+#if BUILDFLAG(ENABLE_IPFS_INTERNALS_WEBUI)
+#define kChromeUIBlobInternalsHost kChromeUIBlobInternalsHost, kIPFSWebUIHost
+#endif
+
 #include "src/chrome/common/webui_url_constants.cc"
 
+#if BUILDFLAG(ENABLE_IPFS_INTERNALS_WEBUI)
```

Comment:
I have recently found out that `#undef` is a no-op if a define doesn't exist, so we don't need to wrap `#undef` declarations into  `#if BUILDFLAG` and similar conditions.

Reply:
Thanks, I didn't know this and I was sure it would give the error.

Fixed at https://github.com/brave/brave-core/pull/17999/commits/21aefb75ac7fc3e6ee47648d3cf20b70311b2a5d, it looks better now.